be/src/cloud/cloud_meta_mgr.cpp

```diff
@@ -711,6 +724,89 @@ Status CloudMetaMgr::sync_tablet_rowsets_unlocked(CloudTablet* tablet,
                         .tag("new_rowsets(rowset,count,cardinality)",
                              fmt::format("[{}]", fmt::join(new_rowset_msgs, ", ")));
             }
+            if (check_delete_bitmap_v2) {
```

Comment:
The code in this branch only depends on old_max_version and resp, so we'd better move this branch to a independent function.